test/e2e/network/service.go

```diff
@@ -1217,6 +1217,28 @@ var _ = SIGDescribe("Services", func() {
 		framework.ExpectNoError(err)
 	})
 
+	ginkgo.It("should be able to create a functioning ClusterIP service with ExternalIPs", func() {
+		serviceName := "externalip-test"
+		ns := f.Namespace.Name
+
+		jig := e2eservice.NewTestJig(cs, ns, serviceName)
+
+		ginkgo.By("creating service " + serviceName + " with type=clusterIP in namespace " + ns)
+		clusterIPService, err := jig.CreateTCPService(func(svc *v1.Service) {
+			svc.Spec.Type = v1.ServiceTypeClusterIP
+			svc.Spec.ExternalIPs = []string{"203.0.113.250","::ffff:cb00:71fa"}
```

Comment:
from the time being, we only have single family cluster and we should use this trick to use the corresponding IP

https://github.com/kubernetes/kubernetes/blob/8058942aa2552bcaf13208d1ea678e9f3f5f4605/test/e2e/network/service.go#L3963-L3965
```go
		externalIP := "203.0.113.250"
		if framework.TestContext.ClusterIsIPv6() {
			externalIP = "2001:DB8::cb00:71fa"
		}
                svc.Spec.ExternalIPs = []string{externalIP}
```